agent/bench-scripts/postprocess/linpack-postprocess

```diff
@@ -29,105 +30,100 @@ my %throughput;
 my @gflops;
 
 my @client_hostname_dirs;
-
 if (opendir(my $clients_dh, "$dir/clients")) {
-    @client_hostname_dirs = grep(!/^\./, (sort  readdir($clients_dh)));
+    @client_hostname_dirs = grep(!/^\./, (sort readdir($clients_dh)));
     closedir($clients_dh);
 } else {
     printf STDERR "ERROR: Could not open clients directory!\n";
     exit 1;
 }
 
 my %client_data;
-
 foreach my $client_hostname_dir (@client_hostname_dirs) {
     $client_data{$client_hostname_dir} = {};
 
     my @client_data_files;
     if (opendir(my $clients_dh, "$dir/clients/$client_hostname_dir")) {
-	@client_data_files = grep(/^linpack\..+$/, (sort readdir($clients_dh)));
```

Comment:
We may have a tabs vs spaces thing going on here.

Reply:
This file was apparently using a mixed approach to indentation.  I seem to have converted it to use spaces rather than tabs.  Would you prefer that it used tabs like the other files?

Reply:
My apologies (I thought I did that, but...).

You can have GitHub turn off whitespace differences, which makes that module pretty straightforward to review.

Reply:
As a rule, while the tool helps to deal with whitespace changes, it is not something that we should allow.

Whitespace changes should be made in separate commits, and it is easy to do by inserting a commit that only makes whitespace changes via `git rebase -i` to edit them into the series of commits.

Reply:
Not every tool people use give you that same way of ignoring whitespace.

And in this case, GitHub gratuitously removes the "hide whitespace changes" when switching to conversations that reference whitespace changes.

Reply:
My apologies, again.  I did put the whitespace changes into separate commits for most of the modules which received significant amounts of whitespace change.  Somehow this module escaped that treatment.

Are you satisfied with the final state of this module?  (I.e., if so, could you mark this conversation "resolved"?)

Reply:
Okay, I'll be a softy and let this slide ... ;)